proposal-template.md

```diff
@@ -43,26 +43,61 @@ Three separate groups of users will be involved in this project. The first group
 
 Item | Project Objectives | Project Outputs | Project Outcomes | Measurement Criteria
 ------------ | ------------- | ------------- | ------------- | -------------
-1 | Suggest routes for crews to robots needing repairs or maintenance more optimal than appending each newly reported broken robot to the end of a crew's itinerary | Route-planning assistance software with ability to push notifications and itinerary to crew members' phones | Decrease in distance travelled for maintenance/ repair crews. ie. savings in fuel and travel time. | Measured by comparing solutions to sample problems consisting of 250 randomly generated locations. The solution must outperform the trivial algorithm on over 95% of 20 test cases. Performance is gauged by the sum of the length of all routes.
+1 | Suggest efficient routes for crews to robots needing repairs or maintenance | Route-planning assistance software with ability to push notifications and itinerary to crew members' phones | Decrease in distance travelled for maintenance/ repair crews. ie. savings in fuel and travel time. | Measured by comparing solutions to sample problems consisting of 250 randomly generated locations. The comparison is with an implementation of a 'trivial' algorithm: routing each crew member to the nearest broken robot. The solution must outperform the basis algorithm on over 80% of 20 test cases, and not perform worse than the trivial algorithm on the other test cases. Performance is gauged by the sum of the length of all routes.
```

Comment:
What is the basis algorithm? Is that the same as the trivial one?

Reply:
LOL yes.. same as trivial.. sorry my brain is mush atm